packages/kube-client/lib/resources/GardenerCore.js

```diff
@@ -66,6 +66,10 @@ class Shoot extends mix(GardenerCore).with(NamespaceScoped, Readable, Observable
     return this.create([namespace, name, 'adminkubeconfig'], body, options)
   }
 
+  patchBindingResource (namespace, name, body, options) {
+    return this.mergePatch(namespace, [name, 'binding'], body, options)
```

Comment:
I don't understand why this works because:
https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1beta1/types_shoot.go#L31

Reply:
Of course `Update` means http method `PATCH`.